duckduckgo/iOS
# externalise MPL dependencies in a submodule

Reviewer: Mia or Caine
Asana: https://app.asana.com/0/414235014887631/466646695795380
CC:

**Description**:

Move the MPL dependencies in to their own project and include them via a submodule.

Note that after doing a pull/check out/clone, you'll now have to run `git submodule update --remote --merge`

**Steps to test this PR**:

* On an iOS 9 device, visit evansycles.com and ensure tracking/blocking looks correct
* On an iOS 10 device, visit evansycles.com and ensure tracking/blocking looks correct
* On an iOS 11 device, visit evansycles.com and ensure tracking/blocking looks correct
